src/blocks/homepage-articles/view.php

```diff
@@ -86,7 +86,9 @@ function newspack_blocks_render_block_homepage_articles( $attributes ) {
 				<article <?php echo has_post_thumbnail() ? 'class="post-has-image"' : ''; ?>>
 					<?php if ( has_post_thumbnail() && $attributes['showImage'] ) : ?>
 						<div class="post-thumbnail">
-							<?php the_post_thumbnail( 'large' ); ?>
+							<a href="' . esc_url( get_permalink() ) . '" rel="bookmark">
```

Comment:
Needs to be in the context of PHP open/close tags:

```suggestion
							<a href="<?php esc_url( get_permalink() ); ?>" rel="bookmark">
```

Reply:
Ah cripes - thanks Jeff! This is fixed in 7bc6437.